[Bug]: Got `java.lang.NoClassDefFoundError: org/apache/hadoop/shaded/com/ctc/wstx/io/InputBootstrapper` error when execute sql in terminal

### What happened?

When I tried to execute some SQL in terminal on Hadoop iceberg catalog, I got the following error:
java.lang.NoClassDefFoundError: org/apache/hadoop/shaded/com/ctc/wstx/io/InputBootstrapper

### Affects Versions

0.7.x

### What table formats are you seeing the problem on?

_No response_

### What engines are you seeing the problem on?

_No response_

### How to reproduce

- Create an internal iceberg catalog with Hadoop storage
- Execute some SQL in the terminal

### Relevant log output

```shell
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.apache.amoro.table.TableMetaStore
        at org.apache.amoro.server.terminal.TerminalManager.getCatalogTableMetaStore(TerminalManager.java:293) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at org.apache.amoro.server.terminal.TerminalManager.executeScript(TerminalManager.java:105) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at org.apache.amoro.server.dashboard.controller.TerminalController.executeScript(TerminalController.java:73) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at io.javalin.core.security.SecurityUtil.noopAccessManager(SecurityUtil.kt:20) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet.addHandler$lambda-0(JavalinServlet.kt:96) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet$lifecycle$2$1$1.invoke(JavalinServlet.kt:43) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet$lifecycle$2$1$1.invoke(JavalinServlet.kt:43) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServletHandler.executeNextTask(JavalinServletHandler.kt:99) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServletHandler.queueNextTaskOrFinish$lambda-1(JavalinServletHandler.kt:85) ~[javalin-4.6.8.jar:4.6.8]
        at java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1187) ~[?:?]
        ... 54 more
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoClassDefFoundError: org/apache/hadoop/shaded/com/ctc/wstx/io/InputBootstrapper [in thread "JettyServerThreadPool-40"]
        at org.apache.amoro.table.TableMetaStore.<clinit>(TableMetaStore.java:73) ~[amoro-core-0.7.1-incubating.jar:0.7.1-incubating]
        at org.apache.amoro.server.terminal.TerminalManager.getCatalogTableMetaStore(TerminalManager.java:293) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at org.apache.amoro.server.terminal.TerminalManager.executeScript(TerminalManager.java:105) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at org.apache.amoro.server.dashboard.controller.TerminalController.executeScript(TerminalController.java:73) ~[amoro-ams-server-0.7.1-incubating.jar:0.7.1-incubating]
        at io.javalin.core.security.SecurityUtil.noopAccessManager(SecurityUtil.kt:20) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet.addHandler$lambda-0(JavalinServlet.kt:96) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet$lifecycle$2$1$1.invoke(JavalinServlet.kt:43) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServlet$lifecycle$2$1$1.invoke(JavalinServlet.kt:43) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServletHandler.executeNextTask(JavalinServletHandler.kt:99) ~[javalin-4.6.8.jar:4.6.8]
        at io.javalin.http.JavalinServletHandler.queueNextTaskOrFinish$lambda-1(JavalinServletHandler.kt:85) ~[javalin-4.6.8.jar:4.6.8]
        at java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1187) ~[?:?]
        ... 54 more
```


### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/pom.xml
+++ b/pom.xml
@@ -489,6 +489,12 @@
                 <artifactId>orc-core</artifactId>
                 <version>${orc-core.version}</version>
                 <classifier>nohive</classifier>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-client-api</artifactId>
+                    </exclusion>
+                </exclusions>
             </dependency>
 
             <dependency>

